Makefile

```diff
@@ -74,6 +74,7 @@ DOCBOOK2PDF ?= docbook2pdf
 
 LCOV ?= lcov
 GENHTML ?= genhtml
+SED ?= sed
```

Comment:
Given that all scripts assume GNU sed, I think it is safer to default to `gsed`.